src/mpi/mpi_caf.c

```diff
@@ -2699,7 +2700,7 @@ PREFIX (sendget) (caf_token_t token_s, size_t offset_s, int image_index_s,
           char error_str[error_len];
           strcpy (error_str, "MPI-error: ");
           MPI_Error_string (mpi_error, &error_str[11], &error_len);
-          PREFIX (error_stop_str) (error_str, error_len + 11);
+          error_stop_str (error_str, error_len + 11, false);
```

Comment:
@jabl Hi Janne, don't we need to keep the prefix macro here? Did something change on the GFortran side that makes this no longer the case?

Reply:
I made a new static function that the PREFIX versions wrap. Since it's used internally as well I thought that would be the best way to handle the API difference due to the new quiet argument.

Reply:
Ah, I see now. So the PREFIX version calls the internal version. Thanks for clarifying.